### DISCUSSION TITLE
Heart Rate Trend

### DISCUSSION THREAD

**Discussion by jones139** [OpenSeizureDetector/Android_Pebble_SD#68] [03_Android App]
A user is interested in setting the heart rate alarm levels appropriately so needs to be able to view the heart rate history - maybe make the main graph in the app a 'Fragment' and have multiple fragments with different data displayed that the user can select?

**Comment by chrisroge** [comment]
Would be great if we could setup schedules, and associate configuration thresholds with the schedule.  For example, I would like to have one schedule for heart rate for the hours I expect my son to be awake and another for when I expect for him to be asleep.

**Comment by jones139** [comment]
Yes, this is possible - I am close to publishing a  test version with some slightly different Heart Rate alarms, in particular an adaptive one that will alarm on rapid changes in heart rate and not slow ones - I am sure we could have a time dependent threshold too, but it might be worth you having a look at whether you think the adaptive heart rate alarm might work for you instead?  See https://www.openseizuredetector.org.uk/?p=1977

**Comment by jones139** [comment]
V5, which is live on play store includes heart rate trends, so closing this - can open a specific issue if there is a particular feature that would still be useful.
